CONTRIBUTING.md

```diff
@@ -38,16 +67,14 @@ task start-dev # or task start-dev-tmux if you want to use tmux panes
 
 To create and test workflows, run the examples in the `./examples` directory.
 
-You will need to add the tenant (output from the `task seed-dev` command) to the `.env` file in each example directory. An example `.env` file for the `./examples/simple` directory can be generated via:
+You will need to add the tenant (output from the `task seed-dev` command) to the `.env` file in each example directory. An example `.env` file for the `./examples/simple` directory. You can be generated and add it to the .env file via:
 
 ```sh
-alias get_token='go run ./cmd/hatchet-admin token create --name local --tenant-id 707d0855-80ab-4e1f-a156-f1c4546cbf52'
-
-cat > ./examples/simple/.env <<EOF
+cat >> ./examples/simple/.env <<EOF
 HATCHET_CLIENT_TENANT_ID=707d0855-80ab-4e1f-a156-f1c4546cbf52
 HATCHET_CLIENT_TLS_ROOT_CA_FILE=../../hack/dev/certs/ca.cert
 HATCHET_CLIENT_TLS_SERVER_NAME=cluster
```

Comment:
These shouldn't actually be necessary and we can remove (those certificates aren't used by the server since we use Caddy - we should actually remove all references to these certs if they exist in other places).

Reply:
https://github.com/hatchet-dev/hatchet/pull/2444/commits/91605425d1d03e6241cc50924f17c9e679a7e4d4